async/sqs/sqs_test.go

```diff
@@ -0,0 +1,103 @@
+package sqs
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestNewConfig(t *testing.T) {
```

Comment:
FYI: Amazon provides a set of interfaces in order to simplify the testing of their SDK integration; using these should be pretty straightforward and would allow you to test things like the `Consume()` method: https://github.com/aws/aws-sdk-go/tree/master/service/sqs/sqsiface

Reply:
I did not see that TBH... I will look into this.